mm2src/coins/hd_wallet/confirm_address.rs

```diff
@@ -83,7 +83,7 @@ where
     Task::InProgressStatus: ConfirmAddressStatus,
     Task::UserAction: TryIntoUserAction + Send,
 {
-    async fn confirm_utxo_address(
+    async fn confirm_address(
```

Comment:
I could not find that confirm_address() can work with trezor eth api although GetNewAddressRpcOps is implemented for EthCoin. is this a TODO?

Reply:
Can you please explain this in more details, should I skip this https://github.com/KomodoPlatform/komodo-defi-framework/blob/ffd0c965d8d071e5f9d0e9f08cb5aa1847d3ac71/mm2src/coins/hd_wallet/coin_ops.rs#L240-L243 for EVM coins or implement `confirm_eth_address_with_trezor` function and add it to `confirm_address` implementation.
P.S. No this is not a TODO, Trezor and HD wallet for EVM should completely work after this PR is merged.

Reply:
Yes, I can see that confim_address() implementation calls Self::confirm_utxo_address_with_trezor but I think this implementation also could be called for the eth coin

Reply:
should be fixed here https://github.com/KomodoPlatform/komodo-defi-framework/pull/1979/commits/d9777aff5afdfab54514a07a81c279c2695615fd